rust/main/chains/hyperlane-cosmos/src/providers/grpc.rs

```diff
@@ -281,27 +448,8 @@ impl WasmGrpcProvider {
         let gas_used = self
             .provider
             .call(move |provider| {
-                let tx_bytes_clone = tx_bytes.clone();
-                let future = async move {
-                    let mut client = TxServiceClient::new(provider.channel.clone());
-                    #[allow(deprecated)]
-                    let sim_req = tonic::Request::new(SimulateRequest {
-                        tx: None,
-                        tx_bytes: tx_bytes_clone,
-                    });
-                    let gas_used = client
-                        .simulate(sim_req)
-                        .await
-                        .map_err(ChainCommunicationError::from_other)?
-                        .into_inner()
-                        .gas_info
-                        .ok_or_else(|| {
-                            ChainCommunicationError::from_other_str("gas info not present")
-                        })?
-                        .gas_used;
-
-                    Ok(gas_used)
```

Comment:
nice, these are technically drive-by changes so worth detailing in the PR description

Reply:
good point :+1: